index.html

```diff
@@ -262,6 +262,14 @@ <h3 class="grouping-title">Attributes</h3>
           <h3 class="grouping-title">CSS Custom Properties</h3>
 
           <ul class="list-attribute">
+            <li>
+              <div>--ar-button-display</div>
+              <p>Sets the display property of the AR button. Intended to be used to force the button to be hidden. Defaults to block.</p>
+            </li>
```

Comment:
Should we list the values of the property that are intended to be used? Will any values do something really weird?

Reply:
`none` is implied by the sentence explaining that this is only meant to hide the UI. But, realistically we cannot force folks not to use other values.